contrib/src/openssl/rules.mak

```diff
@@ -107,12 +107,27 @@ export CROSS_SDK=iPhone${IOS_PLATFORM}.sdk
 endif
 
 ifdef HAVE_TVOS
+
 ifeq ($(MY_TARGET_ARCH),arm64)
-OPENSSL_CONFIG_VARS=ios64-cross
```

Comment:
Oh, sorry, i didn't notice it is under `HAVE_TVOS `. Just notice the modification from `OPENSSL_CONFIG_VARS=ios64-cross` to `TVOS_PLATFORM=OS`.